src/content/reference/app-defaulting-validation/index.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Defaulting and Validation of app CRs
+description: How defaulting and validation of app CRs is implemented by app-admission-controller
+weight: 110
+type: page
+last_review_date: 2020-12-16
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+---
+
+## Overview
+
+For Giant Swarm releases using app-operator version 3.0.0 and upwards the
+defaulting and validation logic of [app CRs]({{< relref "/content/reference/cp-k8s-api/apps.application.giantswarm.io.md" >}}) is enabled. This logic is provided by
+[app-admission-controller](https://github.com/giantswarm/app-admission-controller).
```

Comment:
As a reader, I'd like to understand how I can learn which releases use app-operator v3.0.0 or higher. So we can either name them here, or explain where this can be found out (e. g. by checking the release details in the web interface.)

Reply:
Good point thanks. I'll explain how to check in the release details as I'm not sure which releases it will be in yet.